ios/app/app.xcodeproj/project.pbxproj

```diff
@@ -1031,7 +1035,8 @@
 				OTHER_CPLUSPLUSFLAGS = "$(inherited)";
 				OTHER_LDFLAGS = (
 					"$(inherited)",
-					" ",
+					"-Wl",
+					"-ld_classic",
```

Comment:
are these changes necessary?

Reply:
this is to address the expo EAS build issues